src/frontends/onnx/frontend/src/op/dequantize_linear.cpp

```diff
@@ -32,6 +32,10 @@ std::shared_ptr<ov::Node> get_zero_point(const ov::OutputVector& inputs) {
     if (inputs.size() == 3 && !ov::op::util::is_null(inputs[2])) {
         const auto& zero_point = inputs[2];
 
+        if (zero_point.get_element_type() == ov::element::u16) {
```

Comment:
[random spot]
@z71258847, do I understand correctly that GPU Plugin receives the ov::Model with u16 constants converted to some other type? Because as I can see it should map u16 to i32 before constant folding:
https://github.com/openvinotoolkit/openvino/blob/715790f8eb78bbc550a2a9fa86061f80537448dc/src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp#L625-L645

So, were the constants converted from u16 outside the GPU Plugin?